hazelcast/src/main/java/com/hazelcast/internal/management/events/AbstractWanEvent.java

```diff
@@ -35,7 +36,7 @@ public UUID getUuid() {
     @Override
     public JsonObject toJson() {
         JsonObject json = new JsonObject();
-        json.add("uuid", uuid.toString());
+        json.add("uuid", uuid != null ? uuid.toString() : "null");
```

Comment:
Can this be null, as we have a check in the constructor?

Reply:
I removed the null check.